<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>759</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Fira Code</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="l3_btn_search_by_title">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>By Title</string>
    </property>
   </widget>
   <widget class="QPushButton" name="l3_btn_search_by_description">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>By Description</string>
    </property>
   </widget>
   <widget class="QLabel" name="l3_label_search">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>25</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Search:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="l3_btn_search_random">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>180</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Random</string>
    </property>
   </widget>
   <widget class="QLabel" name="l1_label_error">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&quot;films.db&quot; not found!</string>
    </property>
   </widget>
   <widget class="QLabel" name="l1_label_error_message">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>631</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>  If you already have films database,
put it in the same folder as app and
restart app.

  Or you can create database yourself, 
but it will take some time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="l1_btn_create_db">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>510</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Create db</string>
    </property>
   </widget>
   <widget class="QPushButton" name="l1_btn_exit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>510</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="l2_label_log">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>process log:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="l4_btn_back">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>&lt;-back</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="l4_search_line_edit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>621</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="l4_btn_search">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>40</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="l4_label_search_results">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>search results:</string>
    </property>
   </widget>
   <widget class="QLabel" name="l0_label_verdict">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="l0_label_username">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="l0_username_line_edit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="l0_password_line_edit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="l0_btn_login">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>260</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>16</pointsize>
      <italic>false</italic>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="l0_label_splash">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>25</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Please,Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="l0_label_password">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="l0_btn_register">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>16</pointsize>
      <italic>false</italic>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QPushButton" name="l2_btn_start_creating_db">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="l2_btn_back">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;-Back</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="l4_search_res_text_edit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>831</width>
      <height>571</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Fira Code','Fira Code'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="l3_btn_back">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>&lt;-back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="l3_btn_history">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>History</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="l5_history_text_edit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>831</width>
      <height>631</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Fira Code','Fira Code'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="l5_btn_back">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>&lt;-back</string>
    </property>
   </widget>
   <widget class="QLabel" name="l5_label_history">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>History</string>
    </property>
   </widget>
   <widget class="QPushButton" name="l5_btn_clear">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="l2_log_text_edit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>831</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Fira Code','Fira Code'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;If you want to see log, run this file with:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;python create_db.py&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="l2_label_message">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>681</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>Creating database will take around 4-5 hours.
After that it will contain ~35k films.
Please, wait.</string>
    </property>
   </widget>
   <zorder>l5_history_text_edit</zorder>
   <zorder>l4_search_res_text_edit</zorder>
   <zorder>l4_label_search_results</zorder>
   <zorder>l4_search_line_edit</zorder>
   <zorder>l3_btn_search_by_title</zorder>
   <zorder>l3_btn_search_by_description</zorder>
   <zorder>l3_label_search</zorder>
   <zorder>l3_btn_search_random</zorder>
   <zorder>l1_label_error</zorder>
   <zorder>l1_label_error_message</zorder>
   <zorder>l1_btn_create_db</zorder>
   <zorder>l1_btn_exit</zorder>
   <zorder>l2_label_log</zorder>
   <zorder>l4_btn_back</zorder>
   <zorder>l4_btn_search</zorder>
   <zorder>l0_label_verdict</zorder>
   <zorder>l0_label_username</zorder>
   <zorder>l0_username_line_edit</zorder>
   <zorder>l0_password_line_edit</zorder>
   <zorder>l0_btn_login</zorder>
   <zorder>l0_label_splash</zorder>
   <zorder>l0_label_password</zorder>
   <zorder>l0_btn_register</zorder>
   <zorder>l2_btn_start_creating_db</zorder>
   <zorder>l2_btn_back</zorder>
   <zorder>l3_btn_back</zorder>
   <zorder>l3_btn_history</zorder>
   <zorder>l5_btn_back</zorder>
   <zorder>l5_label_history</zorder>
   <zorder>l5_btn_clear</zorder>
   <zorder>l2_log_text_edit</zorder>
   <zorder>l2_label_message</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
